src/coordinator/mod.rs

```diff
@@ -222,11 +222,7 @@ impl Coordinator {
         Ok(())
     }
 
-    async fn read_time_register(
-        &self,
-        inverter: config::Inverter,
-        action: commands::time_register_ops::Action,
-    ) -> Result<()> {
+    async fn read_time_register(&self, inverter: config::Inverter, action: Action) -> Result<()> {
```

Comment:
Bit concerned that moving the `use` up to a higher level makes this a bit ambiguous, the `Action` is a quite specific action for `time_register_ops` but that knowledge is lost here now until you go hunting for the `use`.

Granted, its just about the only `Action` so far, so its probably ok.

Reply:
I don't have a strong opinion on this, happy to change it back

Reply:
A happy compromise is probably to reduce the `use` level by one then refer to it here as `time_register_ops::Action`? Its still shorter than before :)

Reply:
I started to do that, then noticed that we used `commands::read_inputs`, etc, as well, and didn't like the asymmetry :sweat_smile:

I'm fine leaving it as-is just now TBH, we could always have a separate PR to cut the `commands::` out in a consistent way, rather than changing lots of unrelated lines in this one.